views/add_event.hbs

```diff
@@ -1,79 +1,209 @@
-<div class="add-eventContainer">
-  <h3 class="userAddEvent">Add Event</h3>
-
+{{>header}}
   <form class="add-event-form" action="/add-event" method="post">
     <label>Name </label>
-    <input class="form-control" type="text" name="name_english" placeholder="English" required>
-    <input class="form-control" type="text" name="name_arabic" placeholder="Arabic">
+    <input class="" type="text" name="name_english" placeholder="English" required>
+    <input class="arabic-event-name " type="text" name="name_arabic" placeholder="Arabic">
```

Comment:
Wrap the labels around the inputs, and add a margin-bottom to the labels - then you can get rid of `margin-top: .5rem` and `margin-bottom: .5rem` that you've got multiple times throughout the css

Reply:
That margin also needs to be bigger than it currently is - try and match ze design

Reply:
not sure this is a good idea the margin top and bottom are for making a bit of space between the input the labels have nothing to do with it

Reply:
The labels should be attached to the inputs, and they need to be more spaced out for sure

Reply:
if i wrap the labels im going to ruin the css so what if i use the for attribute in the labels ?

Reply:
I think it would be much easier to do that and have a margin-bottom on the labels, but just make it look like the design and I'll merge